content/guides/models/track/config.md

```diff
@@ -9,15 +9,15 @@ title: Configure experiments
 ---
 {{< cta-button colabLink="https://colab.research.google.com/github/wandb/examples/blob/master/colabs/wandb-log/Configs_in_W%26B.ipynb" >}}
 
-Use the `wandb.config` object to save your training configuration such as: 
+Use the `config` property of a Run to save your training configuration such as: 
 - hyperparameter
 - input settings such as the dataset name or model type
 - any other independent variables for your experiments. 
 
-The `wandb.config` attribute makes it easy to analyze your experiments and reproduce your work in the future. You can group by configuration values in the W&B App, compare the settings of different W&B Runs and view how different training configurations affect the output. A Run's `config` attribute is a dictionary-like object, and it can be built from lots of dictionary-like objects.
+The `run.config` property makes it easy to analyze your experiments and reproduce your work in the future. You can group by configuration values in the W&B App, compare the settings of different W&B Runs and view how different training configurations affect the output. A Run's `config` property is a dictionary-like object, and it can be built from lots of dictionary-like objects.
```

Comment:
> compare the settings of different W&B Runs
instead
compare the configuration of different W&B Runs

Reply:
avoid confusion with wandb.Settings?

Reply:
updated!